breakdancer.js

```diff
@@ -94,4 +94,54 @@ export default class Breakdancer {
 
     return 'unknown';
   }
+
+  /**
+   * Compare current and specified properties of given breakpoint
+   * Requires that this.height() and this.width() are implemented
+   *
+   * @param {String} breakpoint to be compared
+   * @param {String} property height or width
+   * @throws {TypeError} If not given breakpoint and property do not exist within the given spec
+   * @returns {Number} different between current and specified properties
+   * @private
```

Comment:
In all honestly, I would say you might as well just ditch `compareHeight` and `compareWidth` and make the `compare` method public and keep that as only API.